scm-ui/ui-components/src/OverviewPageActions.tsx

```diff
@@ -56,12 +57,17 @@ const OverviewPageActions: FC<Props> = ({
 }) => {
   const history = useHistory();
   const location = useLocation();
+  const [t] = useTranslation("commons");
   const [filterValue, setFilterValue] = useState(urls.getQueryStringFromLocation(location) || "");
   const link = createAbsoluteLink(inputLink);
 
   const groupSelector = groups && (
     <div className="column is-flex">
+      <label id="select-namespace" hidden>
+        {t("namespace")}
+      </label>
       <Select
+        ariaLabelledby="select-namespace"
```

Comment:
This component is used for groups and users, too (and maybe by plugins). Therefore we cannot assume, that we always have namespaces here.